chpl/chpl-service/src/main/java/gov/healthit/chpl/dto/UserTestingLabMapDTO.java

```diff
@@ -0,0 +1,59 @@
+package gov.healthit.chpl.dto;
+
+import java.io.Serializable;
+
+import gov.healthit.chpl.auth.dto.UserDTO;
+import gov.healthit.chpl.entity.UserTestingLabMapEntity;
+
+public class UserTestingLabMapDTO implements Serializable {
+    private static final long serialVersionUID = -4237553934302170475L;
+
+    private Long id;
+    private gov.healthit.chpl.auth.dto.UserDTO user;
```

Comment:
Just curious why the whole package name is needed... it looks like it's being imported above.